l10n_be_iso20022_pain/models/account_payment_line.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import re
+from openerp import api, fields, models, _
+from openerp.exceptions import ValidationError
+
+
+class AccountPaymentLine(models.Model):
+
+    _inherit = 'account.payment.line'
+
+    def _get_communication_type(self):
+        """Add BBA Structured Communication Type"""
+        return[('normal', 'Free Communication'),
+               ('bba', 'BBA Structured Communication')]
+
+    communication_type = fields.Selection(
+        selection=_get_communication_type)
```

Comment:
@ThomasBinsfeld you must extend the inherited selection, not replace it. `selection_add` should help.